docs/middleware/third-party.md

```diff
@@ -3,32 +3,43 @@
 Third-party middleware refers to middleware not bundled within the Hono package.
 Most of this middleware leverages external libraries.
 
-- [ArkType validator](https://github.com/honojs/middleware/tree/main/packages/arktype-validator)
+### Authentication
+
 - [Auth.js(Next Auth)](https://github.com/honojs/middleware/tree/main/packages/auth-js)
-- [Bun Transpiler](https://github.com/honojs/middleware/tree/main/packages/bun-transpiler)
 - [Clerk Auth](https://github.com/honojs/middleware/tree/main/packages/clerk-auth)
+- [OAuth Providers](https://github.com/honojs/middleware/tree/main/packages/oauth-providers)
+- [OIDC Auth](https://github.com/honojs/middleware/tree/main/packages/oidc-auth)
+- [Firebase Auth](https://github.com/honojs/middleware/tree/main/packages/firebase-auth)
+- [Verify RSA JWT (JWKS)](https://github.com/wataruoguchi/verify-rsa-jwt-cloudflare-worker)
+
+### Validators
+
+- [ArkType validator](https://github.com/honojs/middleware/tree/main/packages/arktype-validator)
 - [Effect Schema Validator](https://github.com/honojs/middleware/tree/main/packages/effect-validator)
+- [TypeBox Validator](https://github.com/honojs/middleware/tree/main/packages/typebox-validator)
+- [Typia Validator](https://github.com/honojs/middleware/tree/main/packages/typia-validator)
+- [unknownutil Validator](https://github.com/ryoppippi/hono-unknownutil-validator)
+- [Valibot Validator](https://github.com/honojs/middleware/tree/main/packages/valibot-validator)
+- [Zod Validator](https://github.com/honojs/middleware/tree/main/packages/zod-validator)
+
+### OpenAPI
+
+- [Zod OpenAPI](https://github.com/honojs/middleware/tree/main/packages/zod-openapi)
+- [Scalar API Reference](https://github.com/scalar/scalar/tree/main/packages/hono-api-reference)
+- [Swagger UI](https://github.com/honojs/middleware/tree/main/packages/swagger-ui)
+
+### Others
+
+- [Bun Transpiler](https://github.com/honojs/middleware/tree/main/packages/bun-transpiler)
 - [esbuild Transpiler](https://github.com/honojs/middleware/tree/main/packages/esbuild-transpiler)
 - [ESLint Config](https://github.com/honojs/middleware/tree/main/packages/eslint-config)
```

Comment:
This `ESLint Config` is not middleware for developers. We are mainly using it internally. Please remove this line.

Reply:
It was there before my categorization commit, so I didn't want to remove anything.
This is now done in  https://github.com/honojs/website/pull/468/commits/47c05fb955d1acd2e480dbe27664a8243a2d4786

Reply:
@titouv 

Ah, thank you!